gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPicker.kt

```diff
@@ -197,6 +204,13 @@ internal fun AvatarPicker(uiState: AvatarPickerUiState, onEvent: (AvatarPickerEv
                     .fillMaxWidth()
                     .padding(bottom = 10.dp),
             )
+            AnimatedVisibility(nonSelectedAvatarAlertVisible) {
```

Comment:
I wonder if we could modify this. Currently when this banner gets hidden the whole UI is moved up, rather than the top of the bottom sheet just going down. Feels unnatural.

Reply:
I've been playing with the `AnimatedVisibility` params and options. It's not easy to achieve, or at least it's not pretty straightforward. 

I agree it's nice to have, but I don't want to dedicate too much time to this now. What do you think about creating a ticket and returning to it when we have more "free time"?